subsys/audio_module/Kconfig

```
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Audio"
+
+#----------------------------------------------------------------------------#
+menu "Modules"
+
+config AUDIO_MODULE
+	bool "Enable the audio module"
+
+config AUDIO_MODULE_NAME_SIZE
+	int "Maximum size for module naming in characters"
+	default 20
+
+endmenu # Modules
+
+#----------------------------------------------------------------------------#
+module = AUDIO_MODULE
+module-str = Audio modules
```

I think you might need to do: 
```suggestion
module-str = audio_modules
```